add PHP courses (#6002)
EbookFoundation/free-programming-books: courses/free-courses-en.md

* [Laravel 5.8 Tutorial From Scratch](https://www.youtube.com/playlist?list=PLpzy7FIRqpGD0kxI48v8QEVVZd744Phi4) - Coder's Tape  (2019)
* [Learn how to program: PHP](https://www.learnhowtoprogram.com/php) - Epicodus Inc.

> * [Learn PHP from Scratch](https://www.educative.io/courses/learn-php-from-scratch) - Educative

Reviewer: It's free only until lesson 4.

![image](https://user-images.githubusercontent.com/3125580/139741957-868d3d3d-cf29-4651-994c-eb7927779213.png)

Has Educative platform are any free tier?

* [Learn Top Ten Frameworks In PHP By Building Projects](https://www.eduonix.com/courses/Web-Development/learn-top-ten-frameworks-in-php-by-building-projects) - Eduonix Learning Solutions *(account or email address required)*

> * [Object-Oriented PHP For Beginners](https://www.youtube.com/playlist?list=PLFHz2csJcgk-7hgKrjUa_IP5YCLE4vJhV) - Code with Dary

Reviewer: Complete with instructor and using right syntax for platform
Add moved `Shaun Pelling (The Net Ninja)` tutorial due to fix titles
```suggestion
* [Object-Oriented PHP For Beginners](https://www.youtube.com/playlist?list=PLFHz2csJcgk-7hgKrjUa_IP5YCLE4vJhV) - Dary Nazar (Code with Dary)
* [Object Oriented PHP Tutorial](https://www.youtube.com/playlist?list=PL4cUxeGkcC9hNpT-yVAYxNWOmxjxL51Hy) - Shaun Pelling (The Net Ninja)
```

> * [PHP (& MySQL) Course](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gksOX3Kd9KPo-O68ncT05o) - The Net Ninja (Shaun Pelling)

Reviewer: Complete title, instructor and using right syntax for platform

```suggestion
* [PHP (\& MySQL) Tutorial for Beginners](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gksOX3Kd9KPo-O68ncT05o) - Shaun Pelling (The Net Ninja)
```

* [PHP & MySQL Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWrjkpK2zD4TWKWMWVfeYK-b) - The Bad Tutorials (2015)
* [PHP Courses for Beginner](https://www.learn-php.org) - Learn-PHP.org

> * [PHP For Beginners](https://www.youtube.com/playlist?list=PLFHz2csJcgk_fFEWydZJLiXpc9nB1qfpi) - Code with Dary

Reviewer: Complete with instructor and using right syntax for platform
```suggestion
* [PHP for beginners](https://www.youtube.com/playlist?list=PLFHz2csJcgk_fFEWydZJLiXpc9nB1qfpi) - Dary Nazar (Code with Dary)
```

> * [PHP OOP (Object Oriented) Course](https://www.youtube.com/playlist?list=PL4cUxeGkcC9hNpT-yVAYxNWOmxjxL51Hy) - The Net Ninja (Shaun Pelling)

Reviewer: Moved to ....

Patch to remove
```suggestion
```

* [PHP OOP Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhypQiZnYXM7z7-OTkcMgGPh) - Dani Krossing
* [PHP Programming Language Tutorial - Full Course](https://www.youtube.com/watch?v=OK_JCtrrv-c) - freeCodeCamp.org (2018)
* [PHP Tutorial](https://www.tutorialrepublic.com/php-tutorial/) - TutorialRepublic
* [PHP Tutorials](https://www.youtube.com/playlist?list=PL0eyrZgxdwhwBToawjm9faF1ixePexft-) - Dani Krossing

> * [Use PHP to Create an MVC](https://www.youtube.com/watch?v=6ERdu4k62wI) - FreeCodeCamp

Reviewer: > I'm sorry! I'll remove it!

Not needed. LGTM. +6h video with timestamps integrated in timeline

Apply next patch to complete with full title / instructor according to resources syntax
```suggestion
* [Use PHP to Create an MVC Framework - Full Course](https://www.youtube.com/watch?v=6ERdu4k62wI) - Zura Sekhniashvili (freeCodeCamp.org)
```
